src/redux/reducers.js

```diff
@@ -0,0 +1,15 @@
+import { combineReducers } from 'redux'
+import { routerForBrowser } from 'redux-little-router'
+import router from './router'
+import ui from './ui'
+import todos from './todos'
+
+export const CONTEXT = process.env.NODE_ENV === 'development' ? '' : '/todomvc'
+
+const { reducer } = routerForBrowser({ routes: router, basename: CONTEXT })
+
+export default combineReducers({
+  router: reducer,
+  todos,
```

Comment:
I think our pattern is good and that we should commit into it : 
 - `ui`
    - router
    - form
    - etc
 - `data` (previously `api`)
    - todos

Reply:
👌